SimpleWAWS/Models/ResourceGroup.cs

```diff
@@ -174,7 +179,48 @@ public UIResource UIResource
                     AppService = AppService,
                     TemplateName = templateName,
                     IsExtended = IsExtended,
-                    TimeLeftInSeconds = (int)TimeLeft.TotalSeconds
+                    TimeLeftInSeconds = (int)TimeLeft.TotalSeconds,
+                    CsmId= csmId
+                };
+            }
+        }
+
+        public UIResource FunctionsUIResource
+        {
+            get
+            {
+                string ibizaUrl = null;
+                string csmId = null;
+                Site siteToUseForUi = null;
+                siteToUseForUi = Sites.Where(s => s.IsFunctionsContainer).First();
+                csmId = Sites.Where(s => s.IsFunctionsContainer).First().CsmId;
+                var templateName = Tags.ContainsKey(Constants.TemplateName) ? Tags[Constants.TemplateName] : string.Empty;
+                var appService = AppService.Function;
+                return siteToUseForUi == null
```

Comment:
this can never be true, because `Sites.Where(s => s.IsFunctionsContainer).First()` will throw an exception if there isn't a match